demo/RayPickerCamera/ray_picker_camera.gd

```diff
@@ -49,23 +49,31 @@ func _ready() -> void:
 
     # label all of the cells
     if label_cells == true:
+        var max_neighbour_search_height = 5
+        var label_string = "q:{q}, r:{r}, y:{y}"
+
         for cell_vec in hex_map.get_cell_vecs():
             var cell := HexMapCellId.from_vec(cell_vec)
-
+            var is_top_cell = true
+            var current_search_height = 0
             var above = cell
-            above.y += 1
-            if hex_map.get_cell_item(above) != -1:
+            while is_top_cell && current_search_height <= max_neighbour_search_height:
```

Comment:
Looks like I could use `get_neighours` for this, specifying a plane and distance. Will look at fixing that.